api/turing/service/ensembling_job_service.go

```diff
@@ -180,6 +188,7 @@ func (s *ensemblingJobService) CreateEnsemblingJob(
 	job.JobConfig.Spec.Ensembler.Uri = getEnsemblerDirectory(ensembler)
 	job.InfraConfig.ArtifactURI = ensembler.ArtifactURI
 	job.InfraConfig.EnsemblerName = ensembler.Name
+	job.DashboardURL = s.dashboardURL
```

Comment:
I would've expected that this URL is formatted with the project / namespace / job info for the specific job. Do you have an example of how and where this URL is expected to be used?

For the real time routers, we set the templated URL in the UI config which is populated according to the router being viewed. We've had some discussions in the past, for the real-time router dashboard URL to be moved to the API layer as well (see task id 634 in our backlog), so it may also be available to the SDK. So, I agree with adding the ensembling job's dashboard URL to the API. My concern is just with returning the same value for every job - that feels incomplete to me.

Reply:
This makes sense, I think this should be implemented, feels rather weird when I first did it as well. Let me add this.

Reply:
@krithika369 I have made the changes to allow templating.

Reply:
Can we also use `DashboardURL` -> `MonitoringURL` to align it with Merlin ([here](https://github.com/gojek/merlin/blob/805f33fc67ab17f23a85b1b89283b2f12d100fbf/api/models/version_endpoint.go#L38))?